src/components/FaceDetector/FaceDetector.js

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect, useCallback } from 'react';
+import { useAppMessage } from '@daily-co/daily-react';
+import './FaceDetector.css';
+import FaceDetection from '../../utils/facedetection';
+
+export default function FaceDetector({ id, isLocal, videoRef }) {
+  const [detectionEnabled, setDetectionEnabled] = useState(false);
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Is it `detector` that is unused here? Wondering if there's any way not to set that var (alternatively, in some other demos we specify a `_` prefix as an indicator for unused vars that won't be flagged as well)